README.md

```diff
@@ -1,12 +1,19 @@
 [![GitHub Workflow Status][workflow-shield]][workflow]
-[![GitHub Release][releases-shield]][releases]
+![Codecov](https://img.shields.io/codecov/c/github/DurgNomis-drol/mytoyota)
```

Comment:
All changes to the README should be a separate PR too. This has nothing to do with converting to the new endpoints

Reply:
Please look at it again. The workflow and formatting changes have been rolled back.